### DISCUSSION TITLE
address already in use even after killing the process

### DISCUSSION THREAD

**Discussion by SwapnilSoni1999** [vercel/next.js#73705] [Help]
### Summary

I've been trying to run the nextjs app using the following "start" script:

```json
    "start": "kill-port 1458 && next start -p 1458"
```

but I'm getting the following error 

```sh
Error: listen EADDRINUSE: address already in use :::1458
```

**Comment by icyJoseph** [comment]
What OS do you have?

If in Linux/Mac, try with, `lsof -i :1458`, and then `kill -9` that process. See if that helps first.
